src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -1221,108 +377,133 @@ def main():
+                acquisition_time = normalize_time(
+                    scene["properties"][headers_mapping[eodag_provider]["datetime"]]
+                )
+            except:
+                acquisition_time = scene["properties"][
+                    headers_mapping[eodag_provider]["datetime"]
+                ]
+            product_line += " " + acquisition_time
+            if (
+                REVERSE_PRODUCTTYPE_MAP[scene["properties"]["eodag_product_type"]]
+                in CLOUDCOVER_PRODUCTS
+                and headers_mapping[eodag_provider]["cloud_cover"]
+                in scene["properties"]
+            ):
+                cloud_cover = scene["properties"][
+                    headers_mapping[eodag_provider]["cloud_cover"]
+                ]
+                product_line += f" {cloud_cover:2.0f}%"
+            else:
+                product_line += " cloudcover_NA"
+            product_line += f" {options['producttype']}"
+            print(product_line)
+    else:
 
-    downloader.download(
-        options["output"],
-        options["sleep"],
-        int(options["retry"]),
-        options["datasource"],
-    )
+        if len(scenes["features"]) == 0:
+            gs.warning(_("Nothing to download.\nExiting..."))
+            return
 
+        geojson_temp_dir = gs.tempdir()
+        geojson_temp_file = os.path.join(geojson_temp_dir, "search_result.geojson")
+        with open(geojson_temp_file, "w") as file:
+            file.write(json.dumps(scenes))
+        gs.run_command(
+            "i.eodag",
+            file=geojson_temp_file,
+            provider=eodag_provider,
+            output=outdir,
+            quiet=True,
+        )
     return 0
 
 
 if __name__ == "__main__":
     options, flags = gs.parser()
 
-    if options["datasource"] == "GCS":
-        # Lazy import nonstandard modules
-        try:
-            import pandas
-        except ImportError as e:
-            gs.fatal(_("Module requires pandas library: {}").format(e))
-
-        try:
-            import requests
-        except ImportError as e:
-            gs.fatal(_("Module requires requests library: {}").format(e))
+    try:
+        import eodag
+        from eodag import EODataAccessGateway
+        from eodag.utils.exceptions import AuthenticationError
+    except:
+        gs.fatal(_("Cannot import eodag. Please intall the library first."))
 
-        try:
-            from tqdm import tqdm
-        except ImportError as e:
-            gs.fatal(_("Module requires tqdm library: {}").format(e))
-    elif options["datasource"] == "USGS_EE":
-        try:
-            import landsatxplore.api
-            from landsatxplore.errors import EarthExplorerError
-        except ImportError as e:
-            gs.fatal(_("Module requires landsatxplore library: {}").format(e))
+    try:
+        gs.find_program("i.eodag", "--help")
+    except ImportError:
+        gs.fatal(_("Addon i.eodag not found. Please intall it with g.extension."))
```

Comment:
Shouldn't this be a single message? Users will need to have i.eodag installed which requires eodag.